site/app/templates/admin/Docker.twig

```diff
@@ -5,79 +5,141 @@
     {% if docker_info is not null %}
         {# TODO: include worker information here in the future #}
         <p>Docker Version : {{ docker_info['ServerVersion'] }} </p>
+        <p>Updated on: {{last_updated}}</p>
+        {% if error_logs | length > 0 %}
+            <textarea disabled="disabled" style="resize: none; width: 100%; height:25vh;">
```

Comment:
I think you should put a title above the log display indicating these are logs and not asssociated with the above system information. 

Also HTML should not be used to visually style content, since the output will always be static a textarea is the wrong semantic element to use. You can use a `<pre>` tag which indicates pre-formatted text and will preserve any spacing from the logs and have a similar visual to a textarea's default font.

Reply:
I have made some changes to this area and this is what it looks like now, how does it look?
![image](https://user-images.githubusercontent.com/49821332/127174244-6ac7bd15-df42-4a30-a9b8-e9ccc422b427.png)